examples/phaser/ADAR_pyadi_functions.py

```diff
@@ -0,0 +1,164 @@
+# ADAR_functions.py
+
+import pickle
+
+import numpy as np
+
+# MWT: probably better to pass verbosity to each function
+verbose = True
+
+
+def ADAR_init(beam):
+    # Initialize the ADAR1000
+    beam.reset()  # Performs a soft reset of the device (writes 0x81 to reg 0x00)
+    beam._ctrl.reg_write(
+        0x400, 0x55
+    )  # This trims the LDO value to approx. 1.8V (to the center of its range)
+
+    beam.sequencer_enable = False
+    beam.beam_mem_enable = False  # RAM control vs SPI control of the beam state, reg 0x38, bit 6.  False sets bit high and SPI control
+    beam.bias_mem_enable = False  # RAM control vs SPI control of the bias state, reg 0x38, bit 5.  False sets bit high and SPI control
+    beam.pol_state = False  # Polarity switch state, reg 0x31, bit 0. True outputs -5V, False outputs 0V
+    beam.pol_switch_enable = (
+        False  # Enables switch driver for ADTR1107 switch, reg 0x31, bit 3
+    )
+    beam.tr_source = "spi"  # TR source for chip, reg 0x31 bit 2.  'external' sets bit high, 'spi' sets bit low
+    beam.tr_spi = (
+        "rx"  # TR SPI control, reg 0x31 bit 1.  'tx' sets bit high, 'rx' sets bit low
+    )
+    beam.tr_switch_enable = True  # Switch driver for external switch, reg0x31, bit 4
+    beam.external_tr_polarity = True  # Sets polarity of TR switch compared to TR state of ADAR1000.  True outputs 0V in Rx mode
+
+    beam.rx_vga_enable = True  # Enables Rx VGA, reg 0x2E, bit 0.
+    beam.rx_vm_enable = True  # Enables Rx VGA, reg 0x2E, bit 1.
+    beam.rx_lna_enable = True  # Enables Rx LNA, reg 0x2E, bit 2.
+    beam.rx_lna_bias_current = 8  # Sets the LNA bias to the middle of its range
+    beam.rx_vga_vm_bias_current = 22  # Sets the VGA and vector modulator bias.
+
+
+def ADAR_update_Rx(beam):
+    beam.latch_rx_settings()  # Loads Rx vectors from SPI.  Writes 0x01 to reg 0x28.
+
+
+def ADAR_update_Tx(beam):
+    beam.latch_tx_settings()  # Loads Tx vectors from SPI.  Writes 0x02 to reg 0x28.
+
```

Comment:
These shouldn't exists since they are just a public method rename to another public method

Reply:
Removed. They weren't being called anyway.